payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -24,10 +33,10 @@ internal abstract class BaseAddPaymentMethodFragment(
     protected lateinit var addPaymentMethodHeader: TextView
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
         childFragmentManager.fragmentFactory = AddPaymentMethodsFragmentFactory(
             sheetViewModel::class.java, viewModelFactory
         )
+        super.onCreate(savedInstanceState)
```

Comment:
Seeing the super called after the child fragment is a little weird and might be worth commenting why